src/plugins/plugin-http.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+var shimmer = require('shimmer');
+var url = require('url');
+var isString = require('is').string;
+var merge = require('lodash.merge');
+var httpAgent = require('_http_agent');
+
+function patchModuleRoot (request, api) {
+  var labels = api.labels;
+  function wrapInvocation (options, callback) {
+    return function httpActionTrace () {
+      if (!this._google_trace_patched && options) {
```

Comment:
The original version of the hook doesn't seem to have the `_google_trace_patched` logic. Is this necessary?